miden-remote-provers/build.rs

```diff
@@ -18,13 +18,20 @@ fn main() -> miette::Result<()> {
         return Ok(());
     }
 
-    compile_tonic_server_proto()
+    compile_tonic_client_proto()
```

Comment:
Feel free to disregard this because it should not make any difference, but would this only need to be executed if the `tx-prover` feature is turned on?

Reply:
Not really sure if we get any advantage of not generating the code anyways; I mean, it is not a heavy module. Also, if we build first without the feature and then we add it, we are not going to generate the files, right?